vllm/executor/openvino_executor.py

```diff
@@ -185,6 +189,9 @@ def get_cache_block_size(
         device_config: DeviceConfig,
     ) -> int:
         head_size = model_config.get_head_size()
+        if device_config.device.type == "cpu":
+            if cache_dtype == "u8":
+                head_size += 8
```

Comment:
@luo-cheng2021  Could you please add a comment in separate PR why we need this addition. Worth to describe that quantization params are inlined in the cache.

Reply:
Added in #32.